samples/bluetooth/mesh/ble_peripheral_lbs_coex/prj.conf

```diff
@@ -35,7 +35,7 @@ CONFIG_BT_SETTINGS=y
 CONFIG_BT_RX_STACK_SIZE=2200
 
 CONFIG_BT_EXT_ADV=y
-CONFIG_BT_EXT_ADV_MAX_ADV_SET=2
+CONFIG_BT_EXT_ADV_MAX_ADV_SET=6
```

Comment:
Should we add a small comment on top of this line to show where does number `6` comes from?

May be like this:
```
# Number of Max Adv Sets = peripheral adv (1) + mesh gatt adv (1) 
# + mesh relay (2) ... = 6
```